src/tests/test_journal_vdev.cpp

```diff
@@ -470,6 +503,112 @@ TEST_F(VDevJournalIOTest, Recovery) {
     }
 }
 
+TEST_F(VDevJournalIOTest, MultipleChunkTest) {
+    // Chunk size is 8MB and each data log entry will be of size 3MB to create gaps.
+    uint64_t MB = 1024 * 1024;
+    uint64_t chunk_size = 8 * MB;
+    uint64_t data_size = 3 * MB;
+    JournalDescriptorTest test(1);
+    auto log_dev_jd = test.vdev_jd();
+
+    auto restart_restore = [&]() {
+        test.save();
+        restart_homestore();
+        test.restore();
+        log_dev_jd = test.vdev_jd();
+    };
+
+#define verify_jd(...)                                                                                                 \
```

Comment:
Is there any specific reason it can not be a static function?

Reply:
To keep all the struct members in one line which is easy to read. clang-format splits to multiple lines.

Reply:
c++20 allows that too, right?

Reply:
if we put a function {.a=10,.b=20} , clang-format will split into multiple  5 or 6 lines. Its too fool compiler to keep it one or two line.

Reply:
If clang-format is the concern, we can use below to instruct clang-format not to do that.
// clang-format off
...
// clang-format on

Reply:
Done.